content/geometry/object-environment/environment-general.adoc

```diff
@@ -4,10 +4,10 @@ The environment consists of the terrain, buildings, objects, and traffic areas,
 
 The environment's 3D geometry is structured according to the node structure defined in xref:../geometry/object-environment/environment-index.adoc#_model_structure[Model structure].
 The structure begins with a root node, which is the parent of all other nodes. If applicable, the origin of the root node should match that of the corresponding OpenDRIVE map.
-Children of the root node are terrain, environment objects and the road network.
-The latter is further detailed by its child nodes drivable area, sidewalks, roadmarks, road objects and signals.
+Children of the root node are terrain, environment objects, and the road network.
+The latter is further detailed by its child nodes, the drivable area, sidewalks, roadmarks, road objects, and signals.
 
 An https://github.com/asam-ev/OpenMATERIAL/tree/main/examples/environment_example[example environment asset] is provided in the examples folder.
 
 Groups with a local transform are indicated in the structure by a (T).
-This is only an indicator in the documentation and must not be contained in the actual node name.
\ No newline at end of file
+This is only an indicator in the documentation and is not necessarily contained in the actual node name.
```

Comment:
Why the change to necessarily? This changes the meaning. I would say it must not be contained.

Reply:
Changed back to old text.